watsonsdk/stt/STTConfiguration.m

```diff
@@ -102,6 +102,7 @@ - (NSString *)getStartMessage{
     [inputParameters setValue:self.interimResults forKey:@"interim_results"];
     [inputParameters setValue:self.continuous forKey:@"continuous"];
     [inputParameters setValue:self.inactivityTimeout forKey:@"inactivity_timeout"];
+    [inputParameters setValue:self.learningOptOut forKey:@"x-watson-learning-opt-out"];
```

Comment:
It is TTS and it should be the same as STT. And yes, it says it is query parameter, so it should be like `/v1/recognize?x-watson-learning-opt-out=true` or `/v1/synthesize?x-watson-learning-opt-out=true`, not in the start message and should be before that.

But on Data Collection sections, it says "If you do not want to share your data, set the header parameter X-Watson-Learning-Opt-Out to true for each request. Data is collected for any request that omits this header.", see URLs below:

http://www.ibm.com/watson/developercloud/speech-to-text/api/v1/#data_collection
http://www.ibm.com/watson/developercloud/text-to-speech/api/v1/#data_collection
http://www.ibm.com/watson/developercloud/doc/getting_started/gs-logging.shtml

You can also review https://github.com/watson-developer-cloud/speech-ios-sdk/pull/33/commits/4b20393a83d541c6186db1db11fb7a47038b0e52 for STT and TTS.